media/css/firefox/developer-quantum-whatsnew.scss

```diff
@@ -0,0 +1,181 @@
+        background-position: 0 -169px;
+        padding: 40px 0 60px;
+    }
+
+    .page-title {
+        @include font-size-level2;
+        text-transform: uppercase;
+        margin: 0 auto 20px;
+
+        &:before {
+            display: block;
+            content: '';
+            @include at2x('/media/img/firefox/products/developer/quantum/dev-edition-logo.png', 105px, 108px);
+            margin: 0 auto 20px;
+            width: 105px;
+            height: 108px;
+        }
+    }
+
+    .tagline {
+        max-width: $width-tablet;
+        text-align: center;
+        margin: 0 auto 80px;
+    }
+
+    .features-list {
+        li {
+            margin-bottom: 40px;
+
+            @media #{$mq-tablet} {
+                margin-bottom: 0;
+            }
+        }
+
+        h3 {
+            @include font-size(18px);
+            font-weight: normal;
+            text-transform: uppercase;
+
+            @media #{$mq-phone-wide} {
+                @include font-size(13px);
+            }
+
+            @media #{$mq-desktop} {
+                @include font-size(18px);
+            }
+        }
+
+        a.button.button-hollow {
+            color: $base-blue;
+            border-color: $base-blue;
+            @include font-size-level5;
+            border-radius: 100px;
+            margin-top: 20px;
+
+            @media #{$mq-phone-wide} {
+                padding: .5em 18px;
+                @include font-size(13px);
+            }
+
+            @media #{$mq-desktop} {
+                padding: .5em 36px;
+            }
+        }
+    }
+}
+
+@supports (display: grid) {
+    @media #{$mq-phone-wide} {
+        .gallery-list {
+            grid-gap: 10px 5px;
+            grid-template-columns: repeat(3, 1fr);
+        }
+    }
+}
+
+.section-newsletter {
+    padding: 100px 0;
+
+    .content {
+        padding: 0;
+    }
+
+    .form-title {
+        margin-bottom: 20px;
+        text-align: center;
+
+        @media #{$mq-tablet} {
+            margin-bottom: 0;
+            text-align: right;
+        }
+
+        h3 {
+            @include font-size(18px);
+            text-transform: uppercase;
+
+            @media #{$mq-desktop} {
+                @include font-size(28px);
+            }
+        }
+
+        h4 {
+            @include font-size(14px);
+            font-weight: normal;
+            line-height: $base-line-height;
+
+            @media #{$mq-desktop} {
+                @include font-size(16px);
+            }
+        }
+    }
+}
+
+.section-download {
+    h2 {
+        margin: 0 0 .5em;
+        @include font-size-level2;
+    }
+
+    &:after {
+        content: none;
+    }
+
+    .section-body {
+        @include at2x('/media/img/firefox/template/nightly-logo.png', 165px, 165px);
+
+        @media #{$mq-tablet} {
+            @include background-size(200px 200px);
+        }
+
+        @media #{$mq-desktop} {
+            @include background-size(300px 300px);
```

Comment:
The low-res logo is 180 x 186 so it's being scaled up on non-retina screens and looks blurry. But if we switch to a different graphic we can make it 300 x 300 (and 600 x 600 a 2x).